### DISCUSSION TITLE
How to install NetBox on windows 10 server ?

### DISCUSSION THREAD

**Discussion by DennisHura** [netbox-community/netbox#6090] [Q&A] [answered]
I need to install this program on Windows 10 Server, but i don't know how to do it. I found some guides about how to install Django on windows but some steps i can't repeat. Maybe someone have a guide how to install NetBox on windows

**Comment by ziggekatten** [comment, marked as answer]
You also need Redis, and as there is no suported redis version for windows, going this route is really not an option.

I suggest you run a Linux virtual machine on your Windows server instead. Doing this will make things work out of the box.
